src/components/common/DetailComment.vue

```diff
@@ -16,67 +16,61 @@
   </v-card>
 </template>
 
-<script lang="ts">
-import Vue from 'vue'
-import { Component, Prop, Watch, Emit } from 'vue-property-decorator'
-import { Debounce } from 'vue-debounce-decorator'
+<script setup lang="ts">
+import _ from 'lodash'
+import { computed, ref, watch, defineEmits } from 'vue'
```

Comment:
Does `defineProps` need to be imported here as well?

Reply:
No and defineEmits doesn't need to be there either. Both are part of script setup